internal/tracing/attributes.go

```diff
@@ -1,6 +1,10 @@
 package tracing
 
-import "go.opentelemetry.io/otel/attribute"
+import (
+	arohcpv1alpha1 "github.com/openshift-online/ocm-sdk-go/arohcp/v1alpha1"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+)
```

Comment:
nit: For consistency with other source code files please move the imports below the copyright.